examples/azure/ingress/values.yaml

```diff
@@ -0,0 +1,8 @@
+# This example shows how to deploy and expose GraphDB with Application Gateway Ingress without SSL.
+
+configuration:
+  externalUrl: http://ip.address.nip.io/ # Change this to your AzureDNS domain name or use nip.io which you can do after you deploy the chart since you need to map the public IP address of the Application Gateway to the nip.io.
```

Comment:
The trailing comment is a bit too long. Move it above the property and split it into at least two lines.
Update the other values.yaml as well.